wiki/Community/User_title/en.md

```diff
@@ -43,6 +43,13 @@ On the forum sections of the [old osu! website](https://old.ppy.sh)<!-- TODO: wi
 
 These titles were only visible on the sidebars of forum posts (not on the users' profiles) and were not shown if another custom title was already set prior.
 
+Post count titles were discontinued with the release of the new osu! website.
+
 ## Trivia
 
+::: Infobox
+![](img/multiple-titles.png "VINXIS' two titles being displayed at the same time")
+:::
+
 - User titles are referred to as "rank titles" throughout the game's code. This is a leftover name from when osu! was built on top of [phpBB](https://www.phpbb.com/).
+- As of March 2021, if a user is eligible for multiple user titles, they are displayed together on their profile separated by a slash, instead of only displaying their most recent user title.<!-- VINXIS was the first person to get this treatment when he joined BN, unfortunately the tweet that shows this is deleted -->
```

Comment:
this isn't really true either. lots of people are eligible for multiple titles but don't have them

also, the wording makes it sound like it's a feature of titles themselves, when really this is just a single title assigned manually ... i feel it should say somethign with this idea
```suggestion
- As of March 2021, users earning multiple titles are usually assigned a title that joins all of them together with slashes. Previously, it was more common to display only the most significant or most recent title.
```